pkg/kubelet/cpumanager/cpu_manager_test.go

```diff
@@ -16,48 +16,48 @@ limitations under the License.
 
 package cpumanager
 
-import (
-	"io/ioutil"
-	"reflect"
-	"testing"
-
-	"k8s.io/kubernetes/pkg/kubelet/cpumanager/topo"
-)
-
-func Test_discoverTopology(t *testing.T) {
-	// TODO(CD): Need to provide canned input to make this test portable
-	//           across systems and architectures.
-	cpuInfoFile, err := ioutil.ReadFile("/proc/cpuinfo")
-	if err != nil {
-		t.Errorf("couldn't read /proc/cpuinfo: %v", err)
-		return
-	}
-	type args struct {
-		cpuinfo []byte
-	}
-	tests := []struct {
-		name    string
-		args    args
-		want    *topo.CPUTopology
-		wantErr bool
-	}{
-		{
-			name:    "test",
-			args:    args{cpuinfo: cpuInfoFile},
-			want:    &topo.CPUTopology{NumCPUs: 4, Hyperthreading: false},
-			wantErr: false,
-		},
-	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			got, err := discoverTopology(tt.args.cpuinfo)
-			if (err != nil) != tt.wantErr {
-				t.Errorf("discoverTopology() error = %v, wantErr %v", err, tt.wantErr)
-				return
-			}
-			if !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("discoverTopology() = %v, want %v", got, tt.want)
-			}
-		})
-	}
-}
+//import (
```

Comment:
ah ok, didn't notice Connor had done a copy-paste rather than a rename in that first commit.